WordPress/Classes/ViewRelated/Post/PostEditorAnalyticsSession.swift

```diff
@@ -27,6 +28,10 @@ struct PostEditorAnalyticsSession {
         started = true
     }
 
+    mutating func editorSessionTemplateApplied(_ appliedTemplate: String) {
```

Comment:
The delegate methods use the past tense, in line with typical cocoa conventions, but the session file is using the present tense for other things. I'd suggest using `apply(template:)` for the function and `template` for the instance property.